openslides/mediafiles/views.py

```diff
@@ -15,6 +15,13 @@ class MediafileViewSet(ModelViewSet):
     access_permissions = MediafileAccessPermissions()
     queryset = Mediafile.objects.all()
 
+    def get_queryset(self):
```

Comment:
I don't know but it loosk like that you do not have enough permissions. We changed the permissions system some weeks ago. A suitable migration file is still missing.

Try to remove your personal data directory and then run python manage.py start to create a new settings file and database file.